risc0/zkvm/src/host/receipt.rs

```diff
@@ -269,6 +257,9 @@ impl InnerReceipt {
     }
 
     /// Returns the [InnerReceipt::Flat] arm.
+    ///
+    /// Returns this arm as a slice of [SegmentReceipt]s, as the
+    /// `SegmentReceipts` type used to hold them is a private type.
```

Comment:
I don't think we need to mention that `SegmentReceipts` is a private type, and honestly I think `SegmentReceipts` needs to be a public type.